### PR TITLE
themes and plugins are now copied over into app directory on container start

### PR DESCRIPTION
**Pull request by alex-phillips**
[linuxserverurl]: https://linuxserver.io
[linuxserver.io][linuxserverurl]

### CODE REVIEW

File: root/etc/cont-init.d/50-config [hunk 1]
```diff
@@ -18,11 +18,13 @@ done
 	cp /defaults/rtorrent.rc /config/rtorrent/rtorrent.rc
 cp -pr /config/rutorrent/settings/* /app/rutorrent/conf/
 
-# move and symlink themes folder for custom theme usage
+# copy over any theme files to themes folder
 [[ ! -e /config/themes ]] && \
```
Comment (aptalca) [inline root/etc/cont-init.d/50-config]: I think this should be `[[ -e /config/themes ]]`, is that right?

**Review by aptalca — CHANGES_REQUESTED** [review]

File: root/etc/cont-init.d/50-config (hunk 1 above)
Comment (alex-phillips) [reply root/etc/cont-init.d/50-config]: And that's  why we review :-P

**Review by aptalca — APPROVED** [review]

**Comment by albertusQuelle** [comment]
How does that work @alex-phillips ? I have no folder /config/plugins. 
I would like to add filemanager. But dont know how.

**Comment by Anon0511** [comment]
> How does that work @alex-phillips ? I have no folder /config/plugins.
> I would like to add filemanager. But dont know how.

Hey, you have to create that folder manually in the root of your persistent directory. From there it is the standard like having another folder per plugin. Make sure of the user permissions in there.

Although I think Filemanager did not work when I tested it afterwards since it needs a deeper integration I think. Maybe I missed something. Let me know.

**Comment by dannymichel** [comment]
so you're saying if my ``-v /data/rutorrent:/config`` i should make a folder called ``/data/rutorrent/plugins`` and that plugin should load? why isn't this in the documentation?

**Comment by Anon0511** [comment]
Yeah that is correct. 
I dont know either why it is hidden everywhere.

**Comment by dannymichel** [comment]
> Yeah that is correct.
> I dont know either why it is hidden everywhere.

do you think they have some issue with users using their own plugins?

**Comment by Anon0511** [comment]
No I dont think so. But the Feature to provide your own plugins has been patched in to after I made a request for that. So maybe it just got forgotten or so.

**Comment by Anon0511** [comment]
And dont have the idea to install autodl-irssi, that wont work that easy. I already use a customized fork of this Container with irssi.

**Comment by j0nnymoe** [comment]
You can add in additional software into our containers by doing this: https://blog.linuxserver.io/2019/09/14/customizing-our-containers/
This saves the need to fork.

**Pull request merged**